CMakeLists.txt

```diff
@@ -0,0 +1,114 @@
+cmake_minimum_required (VERSION 3.2)
+project (xtest C)
+
+include(${OPTEE_TEST_SDK}/host_include/conf.cmake)
```

Comment:
use cache entry for user-settable values?
`set(OPTEE_TEST_SDK "<default value>" CACHE STRING "<description>")`

Reply:
I get the principle of what you'd like and agree with that, but I'm not certain how to implement this.
How about taking that after everything is merged?

Reply:
@jenswi-linaro sure